backend.native/tests/build.gradle

```diff
@@ -3340,6 +3340,11 @@ linkTest("inline_lateinitProperty_linkTest") {
     lib = "codegen/inline/lateinitProperty_linkTest_lib.kt"
 }
 
+linkTest("inline_inlineCtor_linkTest") {
```

Comment:
I can't make this test fail when the fix is reverted. Can you?

Reply:
(It looks like I've got tricked by Gradle, let me recheck)

Reply:
Hmm, did you compile with caches on?

Reply:
Well, we don't apply caches for link tests at the moment. I believe we have ticket for this.
Let's keep it as is for now.